api/hubdb.ts

```diff
@@ -11,6 +11,14 @@ import {
 
 const HUBDB_API_PATH = 'cms/v3/hubdb';
 
+export function listTables(
```

Comment:
This is a small nit, but would you mind updating this to be `fetchTables` just to align better with our existing naming ?

Reply:
Good call!